apps/documentation/src/routes/documentation/routing/index.mdx

```diff
@@ -1,15 +1,13 @@
-import { Head } from 'tuono'
+import MetaTags from '../../../components/meta-tags'
 
-<Head>
-  <title>Tuono - Routing</title>
-  <link rel="canonical" href="https://tuono.dev/documentation/routing" />
-</Head>
+<MetaTags
+  title="Tuono - Routing"
+  canonical="https://tuono.dev/documentation/routing/intro"
+/>
 
 import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
 
-<Breadcrumbs>
-  <Element label="Routing" />
-</Breadcrumbs>
+<Breadcrumbs breadcrumbs={[{ label: 'Routing' }]} />
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Don’t put a space after the opening parenthesis. (COMMA_PARENTHESIS_WHITESPACE)
Suggestions: `{`
Rule: https://community.languagetool.org/rule/show/COMMA_PARENTHESIS_WHITESPACE?lang=en-US
Category: TYPOGRAPHY
<!-- __reviewdog__:ChA1MmU2NzUwMTgxMjVhYzVkEgxMYW5ndWFnZVRvb2w= -->